comp/venue/organise-screens.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Organise display screens for the $SRYYYY competition
+
+priority: should
+
+component: Arena
+
+milestone: $SRYYYY Competition
+
+area-owner: event-logistics
+
+description: >-
+  SR owns 6 matching screens, without stands, which are used for the
+  arena displays. These have the required cables to plug into HDMI sources.
+
+  We also have some 19" screens with stands and a variety of other input types.
+
+  We need to figure out what screens we need for the competition and
+  whether we have enough of the right type.
+
+  For the past few years we have used:
+   - 5 screens at the arena (4 corners & staging).
+   - 2/3 screens for the livestream.
+   - 1 screen for scoring entry and comp ops.
+   - Installed screens at the venue presenting at least 1 outside screen and 1 livestream per area of the venue.
+
+  For each screen we will need something to drive it.
+  For the livestream and scoring entry screens these are people's laptops.
```

Comment:
I don't think we should encode that we rely on individuals laptops for these things. Ideally SR should aim to be providing things, even if we do that by asking individuals to lend their devices for the duration.

While the difference here is subtle, I think it's important -- that SR is taking responsibility for these devices for the competition and that SR is explicitly sourcing these rather than relying on people happening to bring suitable devices.